projects/job-tracker.md

```diff
@@ -23,4 +23,9 @@ Srinivasa Teja, Sindhu Banavara, Bhagyasri Canumalla
 
 ### Repository
 
-* [Bitbucket](https://bitbucket.org/nsteja1/5status/src)
+* Source code hosted on [Bitbucket](https://bitbucket.org/nsteja1/5status/src) - Invitations are with Karn, Dblock and Adam.
+* [Jenkins Center](http://5status.com:8080/)  - Unsecured open to Cornell Tech's IP Address. Please let us know if you need some other ipaddress to access
+* [Iam access](https://454863676913.signin.aws.amazon.com/console)
+* [Live Project](http://5status.com/5status/src/index.html)
+* [API](http://5status.com/5status/src/api/v1/<endpoint>)
```

Comment:
​I don't understand, why doesn't it work? Do you mean one of the links
don't work?​
ᐧ

Regards,
Bhagyasri Canumalla

On Wed, Oct 7, 2015 at 3:36 PM, Daniel Doubrovkine (dB.) @dblockdotorg <
notifications@github.com> wrote:

> In projects/job-tracker.md
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/271#discussion_r41435558>
> :
>
> > @@ -23,4 +23,9 @@ Srinivasa Teja, Sindhu Banavara, Bhagyasri Canumalla
> >
> >  ### Repository
> >
> > -* [Bitbucket](https://bitbucket.org/nsteja1/5status/src)
> > +* Source code hosted on [Bitbucket](https://bitbucket.org/nsteja1/5status/src) - Invitations are with Karn, Dblock and Adam.
> > +* [Jenkins Center](http://5status.com:8080/)  - Unsecured open to Cornell Tech's IP Address. Please let us know if you need some other ipaddress to access
> > +* [Iam access](https://454863676913.signin.aws.amazon.com/console)
> > +* [Live Project](http://5status.com/5status/src/index.html)
> > +* [API](http://5status.com/5status/src/api/v1/<endpoint>)
>
> This one doesn't work :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/271/files#r41435558>
> .
>